docker-testing/in-container-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This is the test script that runs in the containers themselves.
+
+WEST=/west
+
+die() {
+    if [ ! -z "$@" ]; then
+        echo "error: $@" >&2
+    else
+        echo "error: unknown error in $0"
+    fi
+    exit 1
+}
+
+main()
+{
+    # Verify the container environment set up meets this script's requirements.
+    [ ! -z "$WEST_TOX_OUT" ] || die "missing $WEST_TOX_OUT"
```

Comment:
```suggestion
    [ -n "$WEST_TOX_OUT" ] || die "missing $WEST_TOX_OUT"
```

The fewer negations the better and copy/paste of `!` can trigger history recall in interactive testing on the command line.